<ui version="4.0" >
 <class>RocketOptionsDialog</class>
 <widget class="QDialog" name="RocketOptionsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Interface</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="chkAlwaysMakeThumbnails" >
         <property name="text" >
          <string>Make thumbnails for large images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkAskBeforeDeleting" >
         <property name="text" >
          <string>Ask before deleting images</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkUseFading" >
         <property name="text" >
          <string>Use subtle fading effects</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkShowRollover" >
         <property name="text" >
          <string>Show rollover menus on thumbnails</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkCenterOnSelection" >
         <property name="text" >
          <string>Center thumbnail toolbox on selected image</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Ad&amp;vanced</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Undo/Redo</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" colspan="3" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>If you have enough RAM, increasing this value will make the program faster.</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QSpinBox" name="spnRamForImages" >
            <property name="suffix" >
             <string> MB</string>
            </property>
            <property name="maximum" >
             <number>256</number>
            </property>
            <property name="minimum" >
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>RAM to be used for storing images:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Other</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkCheckForUpdates" >
            <property name="text" >
             <string>Check for updates when starting</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>211</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>&amp;Watermark</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Wat&amp;ermark</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="1" >
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="1" >
             <widget class="QPushButton" name="btnFont" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>&amp;Font</string>
              </property>
             </widget>
            </item>
            <item rowspan="3" row="0" column="0" >
             <widget class="QTextEdit" name="txtWatermarkText" />
            </item>
            <item row="1" column="1" >
             <widget class="QPushButton" name="btnColor" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>&amp;Color</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>41</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item rowspan="2" row="1" column="0" colspan="2" >
           <widget class="Line" name="line" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radWatermarkText" >
              <property name="text" >
               <string>&amp;Text</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item rowspan="2" row="2" column="1" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLineEdit" name="linImage" >
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnImageSelect" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0" >
           <widget class="QRadioButton" name="radImage" >
            <property name="text" >
             <string>Ima&amp;ge</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>&amp;Opacity</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QSlider" name="sldOpacity" >
            <property name="maximum" >
             <number>255</number>
            </property>
            <property name="pageStep" >
             <number>5</number>
            </property>
            <property name="value" >
             <number>255</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition" >
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval" >
             <number>51</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2" >
        <widget class="QCheckBox" name="chkAddWatermark" >
         <property name="text" >
          <string>&amp;Add watermark to images</string>
         </property>
        </widget>
       </item>
       <item rowspan="2" row="2" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>&amp;Position</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item rowspan="2" row="0" column="3" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>&amp;Margin</string>
            </property>
            <property name="buddy" >
             <cstring>spnMargin</cstring>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="2" column="3" >
           <widget class="QSpinBox" name="spnMargin" >
            <property name="toolTip" >
             <string>Distance from the edge of the image to the text</string>
            </property>
            <property name="maximum" >
             <number>1024</number>
            </property>
            <property name="value" >
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QRadioButton" name="radPosition7" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="2" >
           <widget class="QRadioButton" name="radPosition9" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QRadioButton" name="radPosition8" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="1" column="1" >
           <widget class="QRadioButton" name="radPosition5" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="1" column="2" >
           <widget class="QRadioButton" name="radPosition6" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="1" column="0" >
           <widget class="QRadioButton" name="radPosition4" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QRadioButton" name="radPosition2" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QRadioButton" name="radPosition1" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QRadioButton" name="radPosition3" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QPushButton" name="btnPreview" >
         <property name="text" >
          <string>P&amp;review</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkAlwaysMakeThumbnails</tabstop>
  <tabstop>chkAskBeforeDeleting</tabstop>
  <tabstop>chkUseFading</tabstop>
  <tabstop>chkShowRollover</tabstop>
  <tabstop>chkCenterOnSelection</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>spnRamForImages</tabstop>
  <tabstop>chkCheckForUpdates</tabstop>
  <tabstop>chkAddWatermark</tabstop>
  <tabstop>radWatermarkText</tabstop>
  <tabstop>txtWatermarkText</tabstop>
  <tabstop>btnFont</tabstop>
  <tabstop>btnColor</tabstop>
  <tabstop>radImage</tabstop>
  <tabstop>linImage</tabstop>
  <tabstop>btnImageSelect</tabstop>
  <tabstop>radPosition1</tabstop>
  <tabstop>radPosition2</tabstop>
  <tabstop>radPosition3</tabstop>
  <tabstop>radPosition4</tabstop>
  <tabstop>radPosition5</tabstop>
  <tabstop>radPosition6</tabstop>
  <tabstop>radPosition7</tabstop>
  <tabstop>radPosition8</tabstop>
  <tabstop>radPosition9</tabstop>
  <tabstop>spnMargin</tabstop>
  <tabstop>sldOpacity</tabstop>
  <tabstop>btnPreview</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>txtWatermarkText</sender>
   <signal>textChanged()</signal>
   <receiver>radWatermarkText</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>110</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel" >
     <x>60</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RocketOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>435</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel" >
     <x>540</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RocketOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>392</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel" >
     <x>542</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
